File: Iniciante/1019 - Conversão de Tempo/C#/1019 - Conversão de Tempo.cs
using System;

class URI {
    static void Main() {
        int tempo = Int32.Parse(Console.ReadLine());

        int horas = tempo/3600;
        int minutos = (tempo%3600)/60; // resto de horas dividido por minutos
        int segundos = (tempo%3600)%60; // resto de minutos

        Console.WriteLine("{0}:{1}:{2}", horas, minutos, segundos);

    }
}

File: Iniciante/1019 - Conversão de Tempo/Python/1019 - Conversão de Tempo.py
# entrada do tempo em segundos
N = int(input())

# operações
horas = int(N/3600) # horas = tempo/3600     (3600 == 1 hora)
minutos = int((N%3600)/60) # minutos = resto de horas/60
segundos = int((N%3600)%60) # segundos = resto de minutos/60

# saída do tempo formatado
print('{0}:{1}:{2}'.format(horas, minutos, segundos))